pkg/k8s/option.go

```diff
@@ -0,0 +1,16 @@
+package k8s
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type GetOptions struct {
```

Comment:
I think we should refactor the struct name？GetOptions is not optional, and there is used to create a controller, not get a controller.

Reply:
yes, I will rename GetOptions to CtrlOptions.